src/Locale.php

```diff
@@ -11,11 +11,9 @@ public function __construct()
 
     /**
      * Get absolute Path of default translations.
-     *
-     * @return string
      */
-    public static function getPath()
+    public static function getPath(): string
     {
-        return implode(DIRECTORY_SEPARATOR, [__DIR__, '..', 'locale']) . DIRECTORY_SEPARATOR;
+        return dirname(__DIR__) . '/locale';
```

Comment:
You need to keep `DIRECTORY_SEPARATOR` instead of `/` so I suggest
```
return dirname(__DIR__) . DIRECTORY_SEPARATOR . 'locale';
```

Reply:
for which OS?